assets/styles.css

```diff
@@ -0,0 +1,814 @@
+    margin-bottom: 10px;
+  }
+
+  .grid-img img,
+  .last-grid-img img {
+    width: 75px;
+    height: 80px;
+  }
+
+  .first-section {
+    margin-top: 50px;
+    display: block;
+  }
+
+  .first-section > article {
+    min-width: 300px;
+  }
+
+  .flex-div2 {
+    width: 100%;
+    max-width: 100%;
+    margin: 0;
+    display: block;
+  }
+
+  .small-background-img {
+    width: 100%;
+  }
+
+  .second-headings {
+    display: block;
+  }
+
+  .fas {
+    padding-top: 0;
+  }
+
+  .second-flex-div {
+    margin-bottom: 10px;
+  }
+
+  .second-flex-div i {
+    margin-top: 0;
+  }
+
+  .tnw-down i {
+    padding-top: 0;
+  }
+
+  .omitted-p {
+    margin-top: 0;
+  }
+
+  h3 {
+    margin-block-start: 0;
+    margin-block-end: 0;
+    margin-left: 5px;
+    font-size: 1em;
+  }
+
+  p {
+    margin-block-start: 0;
+    margin-block-end: 0;
+    margin-left: 5px;
+  }
+
+  .grid-img {
+    margin-bottom: 10px;
+  }
+
+  .non-flex-section .spiff-background,
+  .non-flex-section .venture-background {
+    margin-right: 0;
+  }
+
+  .last-grid-img h2 {
+    font-size: 0.9em;
+    margin-right: 10px;
+    margin-left: 10px;
+  }
+
+  .tnw-explanations h1 {
+    font-size: 1em;
+  }
+
+  .tnw-img {
+    display: none;
+  }
+
+  .second-headings h1 {
+    margin-bottom: 10px;
+    font-size: 1.1em;
+  }
+
+  .flex-div * {
+    font-size: 1em;
+  }
+
+  .search-button {
+    display: block;
+  }
+
+  .flex-div {
+    flex-direction: column;
+  }
+
+  .flex-div2 .small-background-img {
+    width: 100%;
+    flex: unset;
+    margin-right: 0;
+  }
+
+  .non-flex-section2 {
+    padding-right: 0;
+  }
+
+  .non-flex-section3 h2 {
+    padding-left: 0;
+  }
+
+  .four-grid-container {
+    display: block;
+    margin-left: 0;
+    margin-right: 0;
+    max-width: 100%;
+  }
+}
+
+@media screen and (min-width: 768px) and (max-width: 1025px) {
+  nav {
+    background-image: url(./logo.png);
+    background-size: 14% 100px;
+    background-position: left;
+    background-repeat: no-repeat;
+    background-position-y: top;
+  }
+
+  nav > :nth-child(1),
+  nav > :nth-child(2) {
+    margin-left: 14%;
+  }
+
+  .flex-div2 {
+    max-width: 100%;
+    margin: 0;
+  }
+
+  .flex-div2 .small-background-img {
+    flex: unset;
+  }
+
+  .four-grid-container {
+    display: grid;
+    grid-gap: 10px;
+    grid-template-columns: repeat(3, 1fr);
+    box-sizing: border-box;
+    max-width: 100%;
+  }
+
+  .four-grid-container h2 {
+    grid-column-start: 1;
+    grid-column-end: 4;
+  }
+
+  .grid-img {
+    width: 100%;
+    height: 100%;
+  }
+
+  .four-grid-container .grid-img img {
+    height: 60%;
+  }
+
+  .flex-div > :nth-child(4) {
+    display: none;
+  }
+
+  .tnw-img {
+    display: none;
+  }
+
+  ul {
+    font-size: 0.6em;
+  }
+
+  ul li a {
+    margin-left: 5px;
+  }
+
+  .flex-container p,
+  .flex-container a {
+    font-size: 0.5em;
+  }
+
+  .second-headings {
+    grid-column-end: 4;
+  }
+
+  #none {
+    display: none;
+  }
```

Comment:
Same style definition of selectors on lines 780 and 790. Group selectors or create a new class.